app/components/Search/SearchPageResults.tsx

```diff
@@ -62,7 +62,7 @@ function SearchResult({ result, onSelect, isSelected }: SearchResultProps) {
         </Link>
         <div>
           {result.content && (
-            <div className={styles.content}>
+            <div>
```

Comment:
hmm, maybe not. But I try not to change the logic of stuff too much here :grimacing:. In case there's some css or something that depends on this being here

Reply:
Fair enough